modules/multiplayer/doc_classes/SceneReplicationConfig.xml

```diff
@@ -66,20 +81,20 @@
 				Sets whether the property identified by the given [param path] is configured to be synchronized on spawn.
 			</description>
 		</method>
-		<method name="property_set_sync">
+		<method name="property_set_sync" is_deprecated="true">
 			<return type="void" />
 			<param index="0" name="path" type="NodePath" />
 			<param index="1" name="enabled" type="bool" />
 			<description>
-				Sets whether the property identified by the given [param path] is configured to be synchronized on process.
+				Sets whether the property identified by the given [param path] is configured to be synchronized on process. See [constant REPLICATION_MODE_ALWAYS].
```

Comment:
For clarity we usually add a deprecated note in the description. E.g.

```suggestion
				Sets whether the property identified by the given [param path] is configured to be synchronized on process.
				[i]Deprecated.[/i] Use [method property_set_replication_mode] with [constant REPLICATION_MODE_ALWAYS] instead.
```

Similar notes can be added to the getters too.